app/views/admin/creator_settings/_form.html.erb

```diff
@@ -13,15 +13,14 @@
     <span class="crayons-field__required crayons-tooltip" data-tooltip="This will set the logo for your Forem" aria-describedby="logo-subtitle"></span>
     <p id="logo-subtitle" class="crayons-field__description">Ideally SVG, but PNG will work, too. Min size: 300x300px.</p>
```

Comment:
Or even "PNG or JPG" rather than "PNG/JPG" with the forward slash! 😊 

I agree with @ridhwana, though, that this shouldn't hold up the PR and can be adjusted in a separate PR.

Reply:
Or even "PNG or JPG" rather than "PNG/JPG" with the forward slash! 😊 

I agree with @ridhwana, though, that this shouldn't hold up the PR and can be adjusted in a separate PR.

Reply:
Or even "PNG or JPG" rather than "PNG/JPG" with the forward slash! 😊 

I agree with @ridhwana, though, that this shouldn't hold up the PR and can be adjusted in a separate PR.

Reply:
I'll mentioned the JPG. Good call. The height of 80px was related to the preview only. I'll omit that part since we'll be auto resizing uploaded logos in a subsequent PR.